common/libs/values.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import Big from 'big.js';
+import { toWei } from 'libs/units';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
any thoughts on if we want to use web3's toHex?

http://web3js.readthedocs.io/en/1.0/web3-utils.html#tohex